### PR TITLE
use sentence style instead of title style

### PR DESCRIPTION
**Pull request by keichwa**
# Description

Short summary of why you created this PR

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [ ] Manager-4.2
- [ ] Uyuni

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/activation-keys.adoc [hunk 1]
```diff
@@ -11,12 +11,12 @@ When the bootstrap script is executed all configuration settings associated with
...
 An activation key can specify:
 
-* Channel Assignment
-* System Types (Traditionally called Add-on Entitlements)
-* Contact Method
-* Configuration Files
-* Packages to be Installed
-* System Group Assignment
+* Channel assignment
+* System types (traditionally called add-on entitlements)
```
Comment (Loquacity) [inline modules/client-configuration/pages/activation-keys.adoc:15]: Do you think that using "traditionally" here might be confusing? Perhaps we could do something like this instead:

```suggestion
* System types (or add-on entitlements)
```
```suggestion
* System types (traditionally called add-on entitlements)
```

**Review by Loquacity — APPROVED** [review]

File: modules/client-configuration/pages/activation-keys.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/activation-keys.adoc:15]: yes, your proposal is better.

**Comment by keichwa** [comment]
And now I'm wondering whether I'm actually allowed to merge this.  I'm asking because of the translations.

**Comment by Loquacity** [comment]
> And now I'm wondering whether I'm actually allowed to merge this. I'm asking because of the translations.

This is minor, go ahead and merge.

**Pull request merged**